cypress/integration/contact_spec.js

```diff
@@ -0,0 +1,85 @@
+describe('Contact Us Page', () => {
+
+  const baseUrl = 'http://localhost:3000'
+
+  before(() => {
+    cy
+    .fixture('peninsulaMockData.json')
+    .then((mockPeninsulaData) => {
+      cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+        statusCode: 200,
+        body: mockPeninsulaData
+      })
+    })
+    cy.visit(baseUrl)
+  });
+
+  it ('Should show the site footer on the main page, containing a link which routes to the Contact page', () => {
+    cy
+      .get('footer').should('be.visible')
+      .find('.contact').should('contain', 'Contact')
+      .should('have.attr', 'href', '/contact-us')
+      .click()
+  })
+
+  it ('Should have the correct url for the Contact page', () => {
+    cy
+      .url().should('include', '/contact-us')
+  });
+
+  it ('Should have a heading for the Contact section', () => {
+    cy
+      .get('.contact-section')
+      .find('h1').should('contain', 'Contact Us')
+  })
+
+  it ('Should have an image of a rowdy tater (Allison)', () => {
+    cy
+      .get('.contacts .AD')
+      .find('.prof-pic').should('have.attr', 'src', '/static/media/AD.8304e665.jpg')
+      .should('have.attr', 'alt', 'BEAUTIFUL ALLISON')
```

Comment:
We may want to change these alt texts to be actually helpful (no judgement on you, I wrote them!)